static/js/tools/map/chart_loader.tsx

```diff
@@ -144,7 +143,9 @@ export function ChartLoader(): JSX.Element {
   useFetchBreadcrumbDenomStat(chartStore, dispatch);
   useFetchMapPointStat(dispatch);
   useFetchGeoRaster(dispatch);
+  useFetchAllDates(dispatch);
 
+  const allSampleDates = useComputeSampledDates(chartStore);
```

Comment:
nit: maybe keep the naming consistent with either "sampleDates" or "sampledDates"

Reply:
Meant to use useComputeSampleDates(). updated